en/docs/develop/restapis/idp.yaml

```diff
@@ -0,0 +1,2311 @@
+      parameters:
+        - name: federated-authenticator-id
+          in: path
+          description: ID of a federated authenticator
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetaFederatedAuthenticator'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-wso2-request: ''
+      x-wso2-response: |
+        "HTTP/1.1 200 Ok"
+  /identity-providers/meta/outbound-provisioning-connectors:
+    get:
+      tags:
+        - Metadata
+      summary: >
+        Metadata about supported outbound provisioning connectors by identity
+        providers in Identity Server
+      description: >-
+        This API provides the list of supported federated authenticators for an
+        IDP in Identity Server
+      operationId: getMetaOutboundConnectors
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/MetaOutboundConnectorListItem'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '500':
+          description: Server Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+      x-wso2-request: ''
```

Comment:
Please don't use these extensions going forward. Use standard swagger keywords, swagger/open-API support capability to define examples. Use the standards.